sql/logic_test.go

```diff
@@ -408,7 +375,7 @@ func (t *logicTest) execQuery(query logicQuery) {
 	}
 	vals := make([]interface{}, len(cols))
```

Comment:
unfortunately not. [`Scan`](http://godoc.org/database/sql#Row.Scan)'s signature is `func (r *Row) Scan(dest ...interface{}) error`, and go does not coerce in this case.